content/news/XXXX-XX-XX-mixxx-2-4-0-features.md

```diff
@@ -0,0 +1,105 @@
+title: Mixxx 2.4 Released
+authors: Mixxx Team
+status: draft
+tags: release, 2.4
+comments: yes
+
+
+We are proud to announce Mixxx 2.4, a new major version of our free open source DJ software, [available right now](https://mixxx.org/download/). It's been two years since the last release due to the large number of great new features and important changes under the hood.
+
+The 2.4 release includes the following highlights:
+
+### EngineOS export
+
+Mixxx 2.4 adds support for exporting crates, playlists and the library to Engine DJ OS devices, including Denon and Numark standalone controllers.
+@Video(https://www.youtube.com/watch?v=Qle9Y1KdD2Y)
+This allows users to prepare their tracks on a laptop using Mixxx and export them to a USB stick for use on a standalone controller at their gig.
+
+### Saved Loops
+
+Mixxx 2.4 adds support for Saved Loops, which can be assigned to any Hot Cue slot.
+If you already have tracks from Serato DJ with Saved Loops, Mixxx 2.4 will import them automatically.
+
+@Video(https://www.youtube.com/watch?v=f8hI_uv9zHY)
+
+### Effect Chains
+
+Mixxx 2.4 introduces Effect Chains, a set of effects which can be saved and reloaded with all their parameters and individual Meta Knob mappings.
+Effect Chains can also be exported and imported, which allows you to share them with other Mixxx DJs.
+Once you've created an Effect Chain, you can create a custom Super Knob mapping which turns the chain into a Quick-Effect, where you take creative control with a single knob.
+Effect parameters can be reordered and or hidden.
+In the effect units, Mixxx 2.4 also displays the value and unit of effect knob parameters while they are being adjusted.
+
+### Track Menu Enhancements
+
+The track menu in Mixxx 2.4 is enhanced by several new features:
+
+- Select loaded track in library.
+- Analyze/re-analyze with variable/constant BPM on a per-track basis.
+- Looking up track metadata at Discogs, Soundcoud and LastFM.
+- Remove track files from disk.
```

Comment:
```suggestion
- Select loaded track in library
- Analyze/re-analyze with variable/constant BPM on a per-track basis
- Looking up track metadata at Discogs, SoundCloud and LastFM
- Remove track files from disk (does 'Move to Trash' with Qt >= 5.15)
```

Reply:
I'd prefer to keep the dots since those are sentences and not short clauses.  If it was a short list of nouns then removing the periods would make sense.